erts/doc/src/erl_dist_protocol.xml

```diff
@@ -741,7 +741,9 @@ io:format("old/unused name ~ts at port ~p, fd = ~p ~n",
 	      node name by setting flag <seeguide marker="#DFLAG_NAME_ME">
 	      <c>DFLAG_NAME_ME</c></seeguide>. The dynamic node name of
 	      <c>A</c> is supplied at the end of the status message from
-	      <c>B</c>.</p>
+	      <c>B</c>. The host name part of <c>Name</c> sent in the
+              <c>send_name</c> message will be used by node <c>B</c> to
+              generate the dynamic node name.</p>
```

Comment:
I think this should be worded something like "The host name of A which was sent as Name in send_name will be used by node B to generate the full dynamic node name."
And it should be added in send_name that Name must be just the host name (without @) when DFLAG_NAME_ME is set.
That is what dist_util:is_name_ok demands: https://github.com/erlang/otp/blob/96872ae52c4461be1791104a511f52ba40afce48/lib/kernel/src/dist_util.erl#L822

Reply:
Thank you for your comment! (those are very important points)
I updated the doc to reflect your review in https://github.com/erlang/otp/pull/5945/commits/c77b25a3ce301043c28e0dd54ddfa0f2c6c61675 and https://github.com/erlang/otp/pull/5945/commits/2b203ad498217502e4ad00c0ba71599dd621a3e1.